src/commands/createContainerApp/TargetPortStep.ts

```diff
@@ -26,6 +26,8 @@ export class TargetPortStep extends AzureWizardPromptStep<IContainerAppContext>
             return localize('enterNumber', 'Enter a valid port number')
         } else if (num < 1 || num > 65535) {
             return localize('portRange', 'Enter a number between 1-65535');
+        } else if (/[^0-9]/.test(val)) {
```

Comment:
```suggestion
        } else if (!Number.isInteger(val)) {
```
And I'd move that check up above the one for under 1 /  over 65535